images/runtimes/css prevent table cell from resizing - Google Search_files/widget_data/cbgapi.js

```diff
@@ -0,0 +1,206 @@
+/* JS */ gapi.loaded_0(function(_){var window=this;
```

Comment:
What are these java scripts about?

Reply:
thanks Carlos, they are not used anywhere, looks like got added by mistake, deleted those.